### ISSUE TITLE
Two issues with `multiselect`

### ISSUE THREAD

**Issue by bobdenotter** [bolt/four#223]
The new "multiselect" items work, but there are some missing bits to make them fully useable: 

```
        <editor-select
            :value="'{{ value }}'".  <- There
            :name="'{{ name }}'"
            :id="'{{ id }}'"
            :options="{{ options|json_encode() }}"
            :form="'{{ form }}'"
        ></editor-select>
```

1) You can pass in a single value like `books` or `movies` and it works. It seems like you can't pass in an array like `["books", "movies"]` to select 2 initially.

2) Selected values don't persist when `POST`ing. You can select items, but these aren't included when the form is submitted

3) There's an odd empty initial selection: 

![screenshot 2018-12-22 at 12 56 48](https://user-images.githubusercontent.com/1833361/50374101-19ce9a80-05e9-11e9-946a-93c0a21df366.png)

@jackiboy Could you take a look at these?

**Comment by bobdenotter** [comment]
I've been working on this.. Second issue is almost fixed. WIP here: https://github.com/bolt/four/compare/working-on-select-for-taxonomies

**Comment by bobdenotter** [comment]
Fixed in #227
